libs/sdk-core/src/models.rs

```diff
@@ -260,8 +260,8 @@ pub struct Payment {
     pub id: String,
     pub payment_type: PaymentType,
     pub payment_time: i64,
-    pub amount_msat: i64,
-    pub fee_msat: i64,
+    pub amount_msat: u64,
```

Comment:
existing wallets will fail to load probably because the fee_msat is stored as negative value in the db.
We need an SQL migration for that.

Reply:
Done, Erdem helped with that.